### DISCUSSION TITLE
Your account has been blocked for suspected abuse. Please contact Support

### DISCUSSION THREAD

**Discussion by 727474430** [community/community#140711] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Your account has been blocked for suspected abuse. Please contact Support

**Comment by jasonxtn** [comment]
Sometimes Github abuse detecting systems highlight accounts that need to be manually reviewed.
You just have to open a ticket : https://support.github.com/
and wait they will fix it .
It happened with me and i got my account back in 10 days.

**Comment by baharbaba** [comment]
I've been waiting for a response for 2 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Comment by CNSeniorious000** [comment]
Me too. @github-staff
